ospfd/ospf_vty.c

```diff
@@ -10526,6 +10526,10 @@ static void show_ip_ospf_route_network(struct vty *vty, struct ospf *ospf,
 								ifindex2ifname(
 									path->ifindex,
 									ospf->vrf_id));
+							json_object_string_addf(
+								json_nexthop,
+								"adv", "%pI4",
```

Comment:
I suggest a more descriptive JSON key name...

Reply:
I have changed it to 'advertiser'.